projects/pt1/python/torch_mlir_e2e_test/test_suite/reduction.py

```diff
@@ -2455,6 +2455,95 @@ def BinaryCrossEntropyWithLogitsStaticModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class HingeEmbeddingLossBasicModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args(
+        [
+            None,
+            ([-1, -1, -1], torch.float32, True),
+            ([-1, -1, -1], torch.float32, True),
+        ]
+    )
+    def forward(self, input, target):
+        return torch.ops.aten.hinge_embedding_loss(
+            input, target, margin=1.5, reduction=1
+        )
+
+
+@register_test_case(module_factory=lambda: HingeEmbeddingLossBasicModule())
+def HingeEmbeddingLossBasicModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(1, 2, 3), tu.rand(1, 2, 3))
+
+
+class HingeEmbeddingLossReductionMeanModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args(
+        [
+            None,
+            ([8, 1], torch.float32, True),
+            ([1, 1], torch.float32, True),
+        ]
+    )
+    def forward(self, input, target):
+        return torch.ops.aten.hinge_embedding_loss(input, target, reduction=1)
+
+
+@register_test_case(module_factory=lambda: HingeEmbeddingLossReductionMeanModule())
+def HingeEmbeddingLossReductionMeanModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(8, 1), tu.rand(1, 1))
+
+
+class HingeEmbeddingLossReductionSumModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args(
+        [
+            None,
+            ([2, 5], torch.float32, True),
+            ([1, 1], torch.float32, True),
+        ]
+    )
+    def forward(self, input, target):
+        return torch.ops.aten.hinge_embedding_loss(input, target, reduction=2)
+
+
+@register_test_case(module_factory=lambda: HingeEmbeddingLossReductionSumModule())
+def HingeEmbeddingLossReductionSumModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(2, 5), tu.rand(1, 1))
+
+
+class HingeEmbeddingLossWithoutReductionModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args(
+        [
+            None,
+            ([8, 5], torch.float32, True),
+            ([1], torch.float32, True),
+        ]
+    )
+    def forward(self, input, target):
+        return torch.ops.aten.hinge_embedding_loss(input, target, margin=1.0)
+
+
+@register_test_case(module_factory=lambda: HingeEmbeddingLossWithoutReductionModule())
+def HingeEmbeddingLossWithoutReductionModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(8, 5), tu.rand(1))
```

Comment:
Please use dynamic shapes in the test unless there is a need for static shape tests to be tested or it's the limitation of the lowering.